HIP/hip-1037.md

```diff
@@ -0,0 +1,95 @@
+---
+hip: 1037
+title: Protocol Buffer API Specification
+author: Joseph Sinclair <@jsync-swirlds>
+requested-by: Richard Bair <@rbair>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Review
+created: 2024-08-09
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1037
+updated: 2024-09-18
+---
+
+## Abstract
+The current protocol buffer API definitions are technically correct, but lack
+usable or useful specification. We should add, in comments compatible with
+protoc-gen-doc, specification text that clearly states both requirement and
+expectation for every message, file, and field. This documentation should be
+automatically transformed to markdown documentation that is published to the
+Hedera API site. In the ideal case this specification will be sufficient for
+any qualified team to implement a completely independent consensus node that
+is fully interoperable with the current consensus node software.
+
+## Motivation
+The existing protocol buffer API documentation is brief, entirely descriptive,
+and often inaccurate. This produces two negative consequences. First, it is not
+possible to produce a compatible independent implementation of the Hedera
+consensus node software without extensive reference to the existing source
+code. Second, developers are often unclear how best to interact with the
+Hedera API, and independent SDK developers, in particular, may struggle to
+correctly implement these API calls.
+As part of expanding the Hedera ecosystem we MUST reduce the difficulty and
```

Comment:
Would read better if this was an actual new paragraph (markdown has thwarted you here, I suspect).  And at L42 as well.

Reply:
I'll add these paragraph breaks (which are just blank lines in markdown).